### DISCUSSION TITLE
Where can i get the list of all the services that are set to manual?

### DISCUSSION THREAD

**Discussion by Tech-Ezralow** [ChrisTitusTech/winutil#795] [Q&A]
Where can i get the list of all the services that are set to manual?
I'm new here, so i was not able to find the list.

**Comment by BourbonCrow** [comment]
Open the winutil.ps1 in this github page and search for
```
function Stop-UnnecessaryServices
```

And you should find the list there

Edit: added link to file
https://github.com/ChrisTitusTech/winutil/blob/main/winutil.ps1
